manifests/bootkube/bootstrap-manifests/kube-controller-manager-pod.yaml

```diff
@@ -0,0 +1,76 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: bootstrap-kube-controller-manager
+  namespace: kube-system
+  labels:
+    openshift.io/control-plane: "true"
+    openshift.io/component: "controller-manager"
+  annotations:
+    openshift.io/run-level: "0"
+spec:
+  restartPolicy: Always
+  hostNetwork: true
+  securityContext:
+    supplementalGroups: [65534]
+  initContainers:
+  - name: setup-lock-dir
+    image: {{ .Image }}
+    command: ["/bin/bash", "-c"]
+    args:
+    - |
+      chgrp 65534 /var/lock
+      chmod 775 /var/lock
+      rm -f /var/lock/* || true
+    securityContext:
+      runAsNonRoot: false
+    volumeMounts:
+    - mountPath: /var/lock
+      name: var-lock
+  containers:
+  - name: kube-controller-manager
+    image: {{ .Image }}
+    imagePullPolicy: {{ .ImagePullPolicy }}
+    command: ["/usr/bin/flock", "--exclusive", "--timeout=60", "/var/lock/controller-manager.lock", "-c"]
+    args:
+    - exec hyperkube kube-controller-manager --config=/etc/kubernetes/config/{{ .ConfigFileName }}
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 65534
+    volumeMounts:
+    - mountPath: /var/lock
+      name: var-lock
+      readOnly: false
+    - mountPath: /etc/ssl/certs
+      name: ssl-certs-host
+      readOnly: true
+    - mountPath: /etc/kubernetes/secrets
+      name: secrets
+      readOnly: true
+    - mountPath: /etc/kubernetes/cloud
+      name: etc-kubernetes-cloud
+      readOnly: true
+    - mountPath: /etc/kubernetes/config
+      name: config
+      readOnly: true
+    livenessProbe:
+      httpGet:
+        scheme: HTTPS
+        port: 8443
```

Comment:
3.11 has no https yet I believe (or did we patch that in?)

Reply:
copied from api server, good catch